features/support/capybara.rb

```diff
@@ -36,6 +36,14 @@
   Capybara.current_driver = :chrome
 end
 
+Before '@security' do
```

Comment:
Is having the same before hook twice intentional?

Reply:
Wow, you are right! 😱 

![image](https://user-images.githubusercontent.com/11318903/81705320-36ce6c80-946f-11ea-8b09-86971e386e6b.png)

![image](https://user-images.githubusercontent.com/11318903/81705351-3f26a780-946f-11ea-9f1c-603ae311bfcb.png)

Thanks for the catch! 🙏

Reply:
hahahahaha I see 😂 Nooope, I wanted to do "Before" and "After", but I see that I do not need any of them 😳

Reply:
Good catch! 😳 🙈